charts/plugin-site/values.yaml

```
@@ -72,9 +72,11 @@ dataFileUrl: https://ci.jenkins.io/job/Infra/job/plugin-site-api/job/generate-da
 github:
   appId: ""
   appPrivateKey: ""
+
 jira:
   username: ""
   password: ""
+
```

```suggestion
jira:
  username: ""
  password: ""
```

Unneeded change (empty lines)